content/en/developers/guide/data-collection-resolution-retention.md

```diff
@@ -10,51 +10,52 @@ algolia:
 
 Find below a summary of Datadog data collection, resolution, and retention:
 
-| Product category          | Source                                           | Collection Methods                                                                                                                    | Collection interval                                                                            | Minimum Resolution    | Default Retention                                                                                                  |
```

Comment:
Not sure why this file looks like so much changed - just added  Cloud Cost Management - GCP line